lib/compiler.js

```diff
@@ -65,6 +65,13 @@ exports.compile = function (code, options) {
     });
 
     importExportVisitor.finalizeHoisting();
+
+    if (getOption(options, "wrapWithRun")) {
+      result.code = "module.run(function(){" +
+        magicString.toString() + "//*/\n});";
+
+      return result;
+    }
```

Comment:
I'm not sure I agree that the REPL should be treated as module code. In particular, I think `this` should still refer to the global object, and the REPL should not force strict mode. Remember the REPL is already a bit weird because we have to generate `var` declarations rather than `let`. I think the goal should be simply to make `import` declarations work in the REPL, with as few other changes as possible.